.prettierrc.json

```diff
@@ -0,0 +1,26 @@
+{
+    "arrowParens": "avoid",
+    "bracketSpacing": true,
+    "jsxSingleQuote": false,
+    "quoteProps": "consistent",
+    "printWidth": 180,
+    "semi": true,
+    "singleQuote": false,
+    "tabWidth": 4,
+    "trailingComma": "all",
+    "useTabs": false,
+    "overrides": [
+        {
+            "files": ["*.html"],
+            "options": {
+                "parser": "go-template"
+            }
+        },
+        {
+            "files": ["*.yml", "*.yaml"],
```

Comment:
It looks like you can reference `.editorconfig` so that you do not have repeat these overrides: https://prettier.io/docs/en/configuration.html#editorconfig

If you're already explored this option, please ignore my comment.